simulators/ethereum/graphql/testcases/48_block_withdrawals.json

```diff
@@ -0,0 +1,23 @@
+{
+  "request":
+  "{ block (number: 33) { number withdrawalsRoot withdrawals { index amount validator address block { hash } } } }",
+
+  "responses": [{
+    "data" : {
+      "block" : {
+        "number" : "0x21",
+        "withdrawalsRoot": "0x37945ab58d2712a26df2a38d217e822694927e29b30d5993d7a53ccea618d1f3",
+        "withdrawals": [{
+          "index": "0x0",
+          "amount": "0x2540be400",
+          "validator": "0xa",
+          "address": "0x0000000000000000000000000000000000000dad",
+          "block": {
```

Comment:
I don't understand why is "block" is added here (or why it works for that matter).

Reply:
I had added it as a back-reference so you can query the block a withdrawal was included in. But thinking about it again, since the only way to get the withdrawals is actually through its block, this is not very useful. Thanks for checking!